### PR TITLE
add cOS

### PR DESCRIPTION
**Pull request by EmmaKnijn**
- [x] I have read the [contribution guidelines](./CONTRIBUTING.md).
- [x] I agree to release my contributions under the [CC-BY-SA](./LICENSE.md) license.
- [ ] I have read the automated feedback from the spell-checker after submitting this pull request. 

## Summary of changes
- Added cOS

### CODE REVIEW

**Comment by EmmaKnijn** [comment]
The linter isn't outputting anything related to my changes?

File: README.md [hunk 1]
```diff
@@ -127,6 +127,8 @@ A note on abbreviations: `CC` is ComputerCraft, `CC:T` is ComputerCraft: Tweaked
     - Phoenix is currently in alpha.
 - [Recrafted](https://github.com/ocawesome101/recrafted) -  A rewrite of CraftOS aiming for full feature parity while maintaining "saner API design."
 - [UnBIOS](https://gist.github.com/MCJack123/42bc69d3757226c966da752df80437dc) -  A program to "undo" CraftOS, returning the system to the same state it was in before running the BIOS. Useful for OS development.
+- [cOS]([https://example.com](https://github.com/knijn/cos)) - the NixOS inspired configurable ComputerCraft OS
```
Comment (tomodachi94) [inline README.md]: Something seems off with the URL here. Also, please add punctuation to the end of the entry for consistency, and capitalize the first "T" in the entry description.

**Review by tomodachi94 — CHANGES_REQUESTED** [review]
Overall looks like a very interesting concept! I've left a few comments. Feel free to ignore the linter, I have a refactor of the CI coming soon.

File: README.md (hunk 1 above)
Comment (EmmaKnijn) [reply README.md]: Should be fixed now. Idk what happened during the pasting of the link

**Comment by tomodachi94** [comment]
Thank you! ❤️

**Pull request merged**